src/main/resources/com/nirima/jenkins/plugins/docker/DockerTemplate/help-remoteFsMapping.html

```diff
@@ -0,0 +1,3 @@
+<div>
+    Enables the ability to browse workspaces of jobs being built using docker containers. Specify the location on the Jenkins master where the job workspaces will be and map them from the images using volumes, network shares, etc.
```

Comment:
maybe "the location on the Jenkins master where the job workspaces will be mapped from the images using volumes, network shares, etc."?